From 9a28caf05b8345cd19276cf7a840599bd9e37749 Mon Sep 17 00:00:00 2001
From: Debbie Martin <Debbie.Martin@arm.com>
Date: Fri, 25 Aug 2023 15:09:33 +0100
Subject: [PATCH] vexpress64: Set the DM_RNG property

Enable the DM_RNG virtio random number generator driver in
in order to consume entropy within u-boot. This is necessary
in the case that the kernel is not configured to enable the
virtio rng driver itself.

Upstream-Status: Pending
Signed-off-by: Debbie Martin <Debbie.Martin@arm.com>
---
 board/armltd/vexpress64/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/board/armltd/vexpress64/Kconfig b/board/armltd/vexpress64/Kconfig
index cf998096e4..7ae5055f59 100644
--- a/board/armltd/vexpress64/Kconfig
+++ b/board/armltd/vexpress64/Kconfig
@@ -21,6 +21,7 @@ config VEXPRESS64_BASE_MODEL
 	imply EFI_SET_TIME if DM_RTC
 	select LINUX_KERNEL_IMAGE_HEADER
 	select POSITION_INDEPENDENT
+	imply DM_RNG

 choice
 	prompt "VExpress64 board variant"
--
2.25.1

